packages/manager/modules/nasha/src/components/partition/create/create.constants.js

```diff
@@ -0,0 +1,5 @@
+export const TRANSLATE = 'nasha_components_partition_create';
```

Comment:
More understanding

```suggestion
export const TRANSLATE_PREFIX = 'nasha_components_partition_create';
```

Reply:
:+1: 
This has been done here : https://github.com/ovh/manager/pull/7495/files#diff-11d14b2a9fc39bc7fa2a726a25d15dda322ebacbe416ed3e98718738340c92c8R1